docs/reference/field-types/relationship.md

```diff
@@ -0,0 +1,119 @@
+# `relationship`
+
+A `relationship` field expresses a one-to-many connection between an Apostrophe document, a piece or page, and another.
+
+The `_id` values for the related docs are stored in an array. When ready to be used, Apostrophe fetches the full related documents, filtering properties as configured, and attaching them on a property matching the `relationship` field name.
+
+For instance, a website may have `pizza` pieces with a `relationship` field named `_toppings` that relates them to `topping` pieces. The topping `_id` values will be stored on `toppingsIds` in the database. In the template and in API responses, the related topping piece data will be available as the `pizza._toppings` array property of each product.
+
+## Module field definition
+
+The field name must begin with an underscore (`_`). This indicates that the ultimate value is not stored in the database, but is populated when needed.
+
+```javascript
+// Configuring the `_toppings` field in a module's `fields.add` subsection:
+_toppings: {
+  label: 'Toppings',
+  type: 'relationship',
+  withType: 'topping',
+  filters: {
+    // Include only the information you need with a projection
+    projection: {
+      title: 1,
+      _url: 1
+    }
+  }
+}
+```
+
+::: warning NOTE
+**For better performance, it is strongly recommended that you set a projection filter** via the `filters` option, limiting the amount of information fetched about each related doc. You may also call other [cursor filters](/reference/modules/apostrophe-docs/server-apostrophe-cursor.md) by setting subproperties of the `filters` property. This is a useful way to limit the acceptable choices for the join.
+:::
+
+## Settings
+
+### Required
+
+|  Property | Type   | Default | Description |
+|-----------|-----------|-----------|-----------|
+|`label` | String | n/a | Sets the visible label for the field in the UI |
+|`type` | String | n/a | Specifies the field type (`relationship` for this type) |
+
+### Optional
+
+|  Property | Type   | Default | Description |
+|-----------|-----------|-----------|-----------|
+|`help` | String | n/a | Help text for the content editor |
+|`htmlHelp` | String | n/a | Help text with support for HTML markup |
+|`filters` | Object | n/a | Provide a list of cursor filters to limit acceptable options for the join. [See below](#filtering-related-document-properties) for more.|
+|`idsStorage` | Object | `{}` | Provide a list of cursor filters to limit acceptable options for the join |
```

Comment:
I think the description here is a copypasta of a draft of the previous one.

Reply:
This is not of type object, it's a string and it's the mongodb property of the doc that will store the ids for the relationship. If the relationship is named `_tacos` it will default to `tacosIds` and using the default is recommended.

Reply:
Yeah, 🍝